---
title: "Untitled"
output: github_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

## Examine Astmha SNPs in the MXL (Mexican Ancestry in LA, California) 1000 Genome sequencing data

```{r}
mxl <- read.csv("373531-SampleGenotypes-Homo_sapiens_Variation_Sample_rs8067378.csv")

mxl
```

Lets focus on 2nd column that contains genotype info

```{r}
genotype <- round(table( mxl[,2] ) / nrow(mxl) * 100, 2)
genotype
```

There are `r genotype[1]` % AA phenotypes in this population.

##Interpreting Base Qualities in R

```{r}

#install.packages("seqinr")
#install.packages("gtools")

library(seqinr)
library(gtools)

phred <- asc( s2c("DDDDCDEDCDDDDBBDDDCC@") ) - 33
phred
```

One sample is obviously not enough to know what is happening in a population. You are interested in assessing genetic differences on a population scale. So, you processed about ~230 samples and did the normalization on a genome level. Now, you want to find whether there is any association of the 4 asthma-associated SNPs (rs8067378...) on ORMDL3 expression.
This is the final file you got ( https://bioboot.github.io/bimm143_S18/class-material/ rs8067378_ENSG00000172057.6.txt ). The first column is sample name, the second column is genotype and the third column is the expression value.

```{r}
# read expression data file
expr <- read.table("https://bioboot.github.io/bimm143_F18/class-material/rs8067378_ENSG00000172057.6.txt")
```

```{r}
inds.aa <- expr$geno =="A/A"
summary(expr$exp[inds.aa])
```

```{r}
inds.ag <- expr$geno =="A/G"
summary(expr$exp[inds.aa])
```

```{r}
inds.gg <- expr$geno =="G/G"
summary(expr$exp[inds.aa])
```

```{r}
#boxplot(count ~ spray. data = InsectSprays, col = "lightgray")
boxplot(exp ~ geno, data=expr)
```

